justfile

```diff
@@ -0,0 +1,55 @@
+# list available commands
+default:
+    @just --list
+
+# Set up local dev environment
+dev_setup:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    . scripts/setup_functions
+    dev_setup
+
+# run the test suite. Optional args are passed to pytest
+test ARGS="":
+    #!/usr/bin/env bash
+    set -euo pipefail
+    . scripts/setup_functions
+    dev_setup
+
+    MODE=slow pytest --cov=cohortextractor --cov=tests {{ ARGS }}
```

Comment:
Can we template the project name here?

If not I don't think it's a great loss, we'll know pretty quickly that it's wrong!

Reply:
Ah, this is a copy/paste error anyway, each project probably wants to set its own cov paths, and they won't necessarily match the project name.  It could just default to `.` which will be over-inclusive, and omit `.venv`

Reply:
Yep that would work 👍